src/connectors/aws_collect.py

```diff
@@ -1372,6 +1394,28 @@
             ],
         },
     },
+    'ebs.describe_snapshot':{
+      'response': {
+            'DescribeSnapshot': {
+                'DataEncryptionKeyID': 'data_encryption_key_id',
+                'KmsKeyID': 'kmskeyid',
```

Comment:
camel case should turn to snake case, i.e. `kms_key_id` here and above

order should be same as in docs